x-pack/plugins/apm/server/lib/alerts/action_variables.ts

```diff
@@ -45,4 +45,24 @@ export const apmActionVariables = {
     ),
     name: 'triggerValue',
   },
+  windowSize: {
+    description: i18n.translate(
+      'xpack.apm.alerts.action_variables.intervalSize',
+      {
+        defaultMessage:
+          'The interval size the alert will use to check for any incident',
```

Comment:
If somebody doesn't understand the term "interval size" it might be a good idea to use a synonym in the description. WDYT of this?

```suggestion
          'The alert will trigger if the conditions are met within the given time period',
```

cc @bmorelli25